diagnostic.md

```diff
@@ -21,13 +21,22 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+curl --include --request "GET" "http://localhost:4741/donuts" \
+  --header "Content-Type: application/json" \
+  --data '{
+    "donuts":
+  }'
 ```
 
 Write an AJAX request to retrieve a list of **all donuts** on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function(id){
+  return $.ajax({
+    url: app.host + '/donuts',
+    method: 'GET',
+  })
+};
 ```
```

Comment:
```js
let getDonuts = function () {
  return $.ajax({
    url: 'http://www.example.com/donuts',
    method: 'GET',
  });
};
```

Don't use app.host here, as we shouldnt assume we have that variable estabilished.